### PR TITLE
Give more meaningful name to siteUtils method

### PR DESCRIPTION
**Pull request by bsclifton**
- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Ran `git rebase -i` to squash commits if needed.

clearSitesWithoutTags => clearHistory

Fixes https://github.com/brave/browser-laptop/issues/3639

Auditors: @bbondy @bridiver 

Test Plan:
- Visit about:history
- Use the "Clear browsing data button"
- Ensure "Browsing history" is on and click "OK"
- Go back to about:history
- Verify history is cleared

### CODE REVIEW

File: js/state/siteUtil.js [hunk 1]
```diff
@@ -371,10 +371,10 @@ module.exports.filterSitesRelativeTo = function (sites, relSite) {
 }
 
 /**
- * Clears out all sites that have no tags.
+ * Clears history (all sites that have no tags).
  * @param sites The application state's Immutable sites list.
  */
-module.exports.clearSitesWithoutTags = function (sites) {
+module.exports.clearHistory = function (sites) {
```
Comment (bridiver) [inline js/state/siteUtil.js]: if this clears history for all sites we should use the method here https://github.com/brave/browser-laptop/blob/master/app/sessionStore.js#L258

File: app/sessionStore.js [hunk 2]
```diff
@@ -254,12 +256,7 @@ module.exports.cleanAppData = (data, isShutdown) => {
   if (data.sites) {
     const clearHistory = isShutdown && getSetting(settings.SHUTDOWN_CLEAR_HISTORY) === true
     if (clearHistory) {
-      // TODO - this should the history methods from siteUtils
-      data.sites = data.sites.filter((site) => site && site.tags && site.tags.length)
-      data.sites = data.sites.map((site) => {
-        delete site.lastAccessedTime
-        return site
-      })
+      data.sites = siteUtil.clearHistory(Immutable.fromJS(data.sites)).toJS()
```
Comment (bsclifton) [inline app/sessionStore.js]: @bridiver per your feedback, I updated this to use the siteUtil method :smile: 

I'm curious what you think about the temp conversion to/from Immutable. I originally made the siteUtil method detect if the input is Immutable, but if feels bad introducing more code paths.

Comment (bridiver) [inline app/sessionStore.js]: I don't think it's a big deal here, it's just a single call in an infrequently called method

**Comment by bridiver** [comment]
can you squash and then I think this is good to merge

**Comment by bsclifton** [comment]
@bridiver squashed :smile:

**Comment by bbondy** [comment]
nice cleanup, thanks.

**Pull request merged**